.github/workflows/release.yml

```diff
@@ -97,3 +97,14 @@ jobs:
           workflow: Docker Hub Release
           repo: twilio/twilio-cli
           token: ${{ secrets.REPO_ACCESS_TOKEN }}
+  platform-executables-release:
+    runs-on: ubuntu-latest
+    needs: [ release ]
+    steps:
+      - name: Invoke Platform Executables workflow
+        if: ${{needs.release.outputs.tag-name != ''}}
+        uses: benc-uk/workflow-dispatch@v1
```

Comment:
Anyway to use something like [this](https://github.com/twilio/twilio-cli-core/blob/a5126f6da0acc7390d4fadc9af8d6192dc451ff1/.github/scripts/trigger-workflow.js#L6) forked functionality in your other PR?

Reply:
Yeah, If one of the PR merged I will update it in another .

Reply:
Sounds good.